src/index.tsx

```diff
@@ -1,13 +1,26 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { Home } from './pages/Home/Home';
+import { People } from './pages/People/People';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <HashRouter>
+  <Routes>
+    <Route path="/" element={<App />}>
+      <Route index element={<Home />} />
+      <Route path="/home" element={<Navigate to="/" replace />} />
+
+      <Route path="/people" element={<People />}>
+        <Route path=":personSlug" element={<People />} />
```

Comment:
The nested route `:personSlug` under `/people` is set to render the `People` component. Verify that this component can handle displaying individual person details based on the `personSlug` parameter.